onnxruntime/test/onnx/testcase_request.cc

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "testcase_request.h"
+#include "dataitem_request.h"
+#include "TestCase.h"
+
+#include "core/common/logging/logging.h"
+#include "core/common/logging/macros.h"
+
+#include <utility>
+
+namespace onnxruntime {
+namespace test {
+
+TestCaseRequestContext::TestCaseRequestContext(const Callback& cb, PThreadPool tp, const ITestCase& c, Ort::Env& env,
+                       const Ort::SessionOptions& session_opts)
+    : cb_(cb),
+      tp_(tp),
+      c_(c),
+      env_(env),
+      session_opts_(session_opts.Clone()),
+      session_(nullptr),
+      allocator_(),
+      result_(),
+      data_tasks_started_(0),
+      data_tasks_inprogress_(0) {
```

Comment:
Shouldn't these be done in the member declarations?

Reply:
I use it to nothing is forgotten and neatly one to another. Generated code is the same

---
In reply to: [488994198](https://github.com/microsoft/onnxruntime/pull/5169#discussion_r488994198) [](ancestors = 488994198)